packages/webapp/webapp_tests.js

```diff
@@ -8,6 +8,24 @@ var hash = crypto.createHash('sha1');
 hash.update(additionalScript);
 var additionalScriptPathname = hash.digest('hex') + ".js";
 
+// convert a stream to a string via promise
+function toString(stream) {
```

Comment:
Alright, this code is duplicated three times now. Let's find a better home for it!

Reply:
If you want to just use an npm package, [`concat-stream`](https://www.npmjs.com/package/concat-stream) looks good to me.